test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -63,18 +62,6 @@ export default class ChallengeOktaVerifyPushPageObject extends ChallengeFactorPa
     return this.form.getErrorBox();
   }
 
-  getErrorTitle() {
-    if (userVariables.gen3) {
-      return this.form.getNthTitle(0);
-    }
-    return this.form.getElement(FORM_INFOBOX_ERROR_TITLE).innerText;
-  }
-
-  getFormTitleWithError() {
-    const titlePosition = userVariables.gen3 ? 1 : 0;
-    return this.form.getNthTitle(titlePosition);
-  }
```

Comment:
getFormTitleWithError() no longer needed since both form title and error title do not share an h2 tag anymore